server/server/src/static_files.rs

```diff
@@ -24,15 +25,24 @@ use service::plugin::plugin_files::{PluginFileService, PluginInfo};
 use service::plugin::validation::ValidatedPluginBucket;
 use service::service_provider::ServiceProvider;
 use service::settings::Settings;
+use service::static_files::StaticFile;
 use service::static_files::{StaticFileCategory, StaticFileService};
 use service::sync::file_sync_driver::get_sync_settings;
 use service::sync::file_synchroniser::FileSynchroniser;
+use service::usize_to_i32;
+use thiserror::Error;
+use util::format_error;
 use util::is_central_server;
-use util::sanitize_filename;
 
 use crate::authentication::validate_cookie_auth;
 use crate::middleware::limit_content_length;
 
+#[derive(Debug, MultipartForm)]
+pub(crate) struct UploadForm {
```

Comment:
this is now used for fridge tag upload and sync on central (we don't have a common place for shared things like this in server crate)